source/instructors/authoring/assessments/multiple-choice.rst

```diff
@@ -5,94 +5,59 @@
 
 Multiple Choice
 ===============
-Multiple choice type assessments provide a question and then single or multiple response options. 
+Multiple choice assessments allow you to present a question with single or multiple response options. These questions can be created manually or generated using AI. 
 
 Assessment Auto-Generation
-++++++++++++++++++++++++++
+--------------------------
 
-Assessments can be auto-generated using the text on the current guides page as context. Follow the steps below to auto-generate a Multiple Choice assessment:
-
-1. Select **Multiple Choice** from the Assessments list.
-
-2. Press the **Generate** button at bottom right corner. 
-
-   .. image:: /img/guides/generate-assessment-button.png
-      :alt: Generate assessment button
-
-3. The Generation Prompt will open, press **Generate Using AI** to preview the generated assessment.
-
-   .. image:: /img/guides/assessment-generation-prompt.png
-      :alt: Assessment Generation Prompt
-
-
- If you are not satisfied with the result, select **Regenerate** to create a new version of the assessment. You can provide additional guidance in the **Generation Prompt** field. For example, *create assessment based on the first paragraph with 2 correct answers.*
-
-4. When you are satisfied with the result, press **Apply** and then **Create**.
-
-
-More information about generating assessments may be found on the :ref:`AI assessment generation <ai-assessment-generation>` page.
+Assessments can be auto-generated using the text on the current guide page as context. For more information, see the :ref:`AI assessment generation <ai-assessment-generation>` page.
 
 
 Assessment Manual Creation
-++++++++++++++++++++++++++
+--------------------------
 
-Follow these steps to set up multiple choice assessments manually:
+Follow these steps to set up multiple choice assessments manually. For more information on **Metadata** and **Files**, see :ref:`Assessments <assessments>`.
```

Comment:
For more information on **General**, **Metadata** (optional) and **Files** (optional) see :ref:`Assessments <assessments>`.